# Define the goal
geometry_msgs/Pose poseR
geometry_msgs/Pose poseL
float32 duration
---
# Define the result
bool reached_goal
---
# Define a feedback message
float32 progress